support/cas-server-support-saml-idp-core/src/main/java/org/apereo/cas/support/saml/services/SamlRegisteredService.java

```diff
@@ -85,6 +85,12 @@
     @Column
     private String issuerEntityId;
 
+    @Column
+    private String issuerSigningKeyLocation;
+
+    @Column
+    private String issuerSigningAlgorithm;
```

Comment:
Change this to "signingKeyAlgorithm".

Reply:
Also, please document the field and its purpose and add a few test cases to demonstrate how the algorithm override may work.